README.md

```diff
@@ -2,6 +2,21 @@
 
 Microservice to load datapoints into the Data Discovery database, driven by Kafka messages.
 
+### Configuration
+
+The port that the application runs on needs to be passed as an option to the JVM `-Dhttp.port=20097`
+When running locally the default port is set in the .sbtops file
+
+| Environment variable | Default | Description
+| -------------------- | ------- | -----------
+| APPLICATION_SECRET   | changeme                                         | Required by Play framework
```

Comment:
No its something baked into the Play framework. I could not see anything obvious to remove the need for it.